test/test_topology.ml

```diff
@@ -0,0 +1,215 @@
+open Topology
+
+module D = Debug.Make (struct
+  let name = "test_topology"
+end)
+
+let make_numa ~numa ~sockets:_ ~cores =
```

Comment:
`sockets` is unused. Will it be used in the future to generate the distance matrix?

Reply:
Initially I had the socket number in the cpu map, but later I used only the numa node, I'll drop this from here.